manuscript/code/hello-world-minified.c

```diff
@@ -1 +1 @@
-uint64_t*foo;uint64_t*main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
\ No newline at end of file
+char*foo;uint64_t main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
```

Comment:
This breaks the original idea of zero whitespace in the minified version...

Reply:
changed it back to `uint64_t*main`